pom.xml

```diff
@@ -11,15 +11,15 @@
     <parent>
         <groupId>org.springframework.boot</groupId>
         <artifactId>spring-boot-starter-parent</artifactId>
-        <version>1.5.14.RELEASE</version>
+        <version>2.3.3.RELEASE</version>
         <relativePath/>
         <!-- lookup parent from repository -->
     </parent>
 
     <properties>
         <java.version>1.8</java.version>
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
-        <hibernate.version>5.3.1.Final</hibernate.version>
+        <hibernate.version>5.4.21.Final</hibernate.version>
```

Comment:
I believe hibernate version is set also in Spring Boot parent POM with all dependencies.

Reply:
Indeed! Fixed.